docker/.gitignore

```diff
@@ -1 +0,0 @@
-data
```

Comment:
Remove this commit, please.

Reply:
When there is `.gitignore` which ignores `data` folder, `data/.gitkeep` would not be tracked in git.

Reply:
```bash
mkdir -p docker/data/clickhouse/
touch docker/data/clickhouse/.gitkeep
git add --force docker/data/clickhouse/.gitkeep
git status
```
Git ignore everything in an ignored directory but will track the added file (and `--force` allows you to skip `.gitignore` check)